packages/@okta/vuepress-site/docs/guides/terraform-enable-org-access/main/index.md

```diff
@@ -26,83 +26,113 @@ Create an Okta app and credentials that Terraform uses to manage the objects in
+1. For a production deployment, securely store the private key in a Terraform secrets management system or other key management system.
 
-   > **Note:** The private key only appears in this dialog once. Losing the private key requires generating a new pair of keys.
-1. Click **Done**, **Save**, and then **Save** again.
-
-> **Note:** Set the status of any public key currently used by Terraform to **Active**, and set the status of the other public keys to **Inactive**.
+### Use an external tool to generate the public/private key pair
 
-Check that the generated private key is in PKCS#1 format, which is the format required by the Okta Terraform Provider. In that format, the file that contains the private key begins with `-----BEGIN RSA PRIVATE KEY-----`.
-
-If the key isn't in the right format, convert it to the correct format using the OpenSSL command line program:
+1. Audit your external tool to generate a key pair for security. Follow organization best practices and audit any code beforehand to maintain org security. Never use an untrusted third-party web site to generate keys for production systems.
+1. Check that the generated private key is in PKCS#1 format, which is the format required by the Okta Terraform provider. In that format, the file that contains the private key begins with `-----BEGIN RSA PRIVATE KEY-----`. If the key isn't in the right format, convert it to the correct format using the OpenSSL command line program:
 
 1. In a terminal, go to the file path where you saved the original private key.
 1. Run OpenSSL to convert the key. One of the following command lines should work depending on your operating system and version of OpenSSL:
-`openssl rsa -in {ORIGINAL_PRIVATE_KEY} -out {CONVERTED_PRIVATE_KEY} -traditional`
-`openssl rsa -in {ORIGINAL_PRIVATE_KEY} -out {CONVERTED_PRIVATE_KEY}`
-`openssl-rsa -in {ORIGINAL_PRIVATE_KEY} -out {CONVERTED_PRIVATE_KEY}`
-
-   * `ORIGINAL_PRIVATE_KEY`: The file that contains the key generated earlier.
-   * `CONVERTED_PRIVATE_KEY`: The file that contains the converted key.
-
-> **Note:** The file that contains the converted private key must begin with `-----BEGIN RSA PRIVATE KEY-----`. If not, try step two again.
+   `openssl rsa -in {ORIGINAL_PRIVATE_KEY} -out {CONVERTED_PRIVATE_KEY} -traditional`
+   `openssl rsa -in {ORIGINAL_PRIVATE_KEY} -out {CONVERTED_PRIVATE_KEY}`
+   `openssl-rsa -in {ORIGINAL_PRIVATE_KEY} -out {CONVERTED_PRIVATE_KEY}`
 
-## Assign admin roles
+   In this command:
+   * `ORIGINAL_PRIVATE_KEY`: Set this to the file path of the original key.
+   * `CONVERTED_PRIVATE_KEY`: Set this to the file path for the converted key.
+1. Confirm the converted key begins with `-----BEGIN RSA PRIVATE KEY-----`. If not, try these steps again.
+1. Save the converted private key locally to somewhere on your system that you can reference it from Terraform. By convention, save the file with a file name ending in `.pem`. 
```

Comment:
This point could be a bit easier to read... perhaps:
"Save the converted private key file. By convention, the file extension for a private key file is `.pem`.

Reply:
```suggestion
1. Save the converted private key file. By convention, the file extension for a private key file is `.pem`. 
```